client/landing/gutenboarding/components/locale-context/index.tsx

```diff
@@ -74,19 +81,15 @@ export const LocaleContext: React.FunctionComponent = ( { children } ) => {
 		}
 	};
 
-	const loadInitalLocale = async () => {
+	async function loadInitalLocale() {
```

Comment:
This change was to fix a style complaint about using a the function before it's declared.